src/core/push_kit.js

```diff
@@ -9,15 +9,20 @@ import { Client } from "@stomp/stompjs";
 
 const push_kit = {
     init() {
-        const pps = $("meta[name=patterns-push-server]").attr("content");
-        const exchange_url = "/exchange/quaive";
-        if (!pps) return;
+        document.querySelector('meta[name="patterns-push-server"]').content
+        const push_url = $("meta[name=patterns-push-server]").attr("content");
+        const push_path = $("meta[name=patterns-push-path]").attr("content");
+        const push_user = $("meta[name=patterns-push-pass-TODO]").attr("content");
```

Comment:
should this not rather be `patterns-push-user-TODO`?
```suggestion
        const push_user = $("meta[name=patterns-push-user-TODO]").attr("content");
```